src/_adr_autocomplete

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
-eval "$($(dirname $0)/adr-config)"
+eval "$("$(dirname $0)"/adr-config)"
```

Comment:
most probably this will produce some issues

Reply:
You're right. will update the PR accordingly.